cherrypy/test/test_http.py

```diff
@@ -160,10 +180,13 @@ def test_post_filename_with_special_characters(self):
         '''Testing that we can handle filenames with special characters. This
         was reported as a bug in:
            https://github.com/cherrypy/cherrypy/issues/1146/
-           https://github.com/cherrypy/cherrypy/issues/1397'''
+           https://github.com/cherrypy/cherrypy/issues/1397
+           https://github.com/cherrypy/cherrypy/issues/1694'''
```

Comment:
It would have been nice if the preceding commit had formatted without the trailing end symbol, but they didn't so now we get a messy diff. Let's not repeat that mistake and instead terminate the multiline string on another line.